deploy/template.yaml

```diff
@@ -1081,7 +1082,7 @@ Resources:
     Type: "AWS::SQS::Queue"
     Properties:
       MessageRetentionPeriod: !FindInMap [EnvironmentVariables, !Ref Environment, MESSAGERETENTIONPERIODDAYS]
-      VisibilityTimeout: 60
+      VisibilityTimeout: 300
```

Comment:
can you take a look at https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#events-sqs-queueconfig - the AWS recommendation is this is at least 6 times the lambda timeout